frontend/website/src/Blog.re

```diff
@@ -26,6 +26,19 @@ let titleColor = c => Css.unsafe("--title-color", Style.Colors.string(c));
 let readMoreColor = c =>
   Css.unsafe("--read-more-color", Style.Colors.string(c));
 
+// Truncate HTML
+let rec truncate = (index, counter, html) =>
```

Comment:
Might be a matter of taste but I tend to reverse the counter in these situations, so that I would pass 5 into the `truncate` and it counts down until 0. That way the user of the function has some more control about how much they want to truncate.

Reply:
makes sense, i agree!